cliquet/listeners/redis.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import absolute_import
+from six.moves.urllib import parse as urlparse
+import redis
+import json
+
+from cliquet.listeners import ListenerBase
+
+
+class RedisListener(ListenerBase):
+    def __init__(self, *args, **kwargs):
+        super(RedisListener, self).__init__(*args, **kwargs)
+        maxconn = kwargs.pop('max_connections')
+        self._listname = kwargs.pop('listname', 'cliquet.events')
+        connection_pool = redis.BlockingConnectionPool(max_connections=maxconn)
+        self._client = redis.StrictRedis(connection_pool=connection_pool,
+                                         **kwargs)
+
+    def __call__(self, event):
+        try:
+            payload = json.dumps(event.payload)
+        except Exception:
```

Comment:
> Yes but since the listeners are executed synchronously it would be equivalent and would apply to every listeners.

Not equivalent. 

For instance if I have a redis listener and an http listener (in that order) and if only the redis listener fails before the redis server is down or something, catching the error at the upper level means that the http listener will never be called. 

That said, adding a try..catch at the upper level is also a good idea